### DISCUSSION TITLE
Set to NULL button cutting off text fields in Table Editor side panel

### DISCUSSION THREAD

**Discussion by saltcod** [supabase/supabase#16988] [Contribute to Supabase]
Long text fields are hidden behind the Set to NULL button when you edit a table row via the side panel in the Table Editor

![CleanShot 2023-08-07 at 10 48 32@2x](https://github.com/supabase/supabase/assets/105593/5478b2ae-6044-48a3-8ace-9e91d3c2217f)

![CleanShot 2023-08-07 at 10 46 59@2x](https://github.com/supabase/supabase/assets/105593/e8ec63e5-d86b-4741-8c60-86dd2369a59b)

**Comment by avi1737** [comment]
can I take this up ?

**Comment by Abdul-Maajith** [comment]
Let Me Try?

**Comment by faizanraso** [comment]
@saltcod I've submitted a PR to resolve this issue https://github.com/supabase/supabase/pull/16351
